docs/submitters/submissionGuide.md

```diff
@@ -0,0 +1,169 @@
+increment the minor version number for beta versions,
+and increment the major version number for stable versions.
+
+Example:
+```json
+{
+	"major": 21,
+	"minor": 6,
+	"patch": 0
+}
+```
+
+### addonVersionName
+The addon version being released.
+Must match the version in the addon manifest and the file name for the submission.
+
+Example: `21.6.0`.
+
+### displayName
+The name that will be displayed in English for the addon.
+
+Example: `"Easy Addon"`
+
+### publisher
+The name of the individual, group, or company responsible for the addon.
+
+Example: `"NV Access"`
+
+### description
+The English description of the addon that will be displayed for the addon.
+	
+Example: `"Makes doing XYZ easier"`
+
+### homepage
+Optional.
+If the addon has a homepage where users can get more information about the addon, you can specify it here.
+
+Example: `"https://github.com/nvaccess/addon-datastore"`
+
+### minNVDAVersion
+The addon will not work with versions of NVDA prior to this version.
+
+Example:
+```json
+{
+	"major": 2021,
+	"minor": 1,
+	"patch": 0
+}
+```
+
+### lastTestedVersion
+The add-on has been tested up to and including this version of NVDA.
+
+Example:
+```json
+{
+	"major": 2020,
+	"minor": 4,
+	"patch": 0
+}
+```
+
+### URL
+To allow directly downloading the *.nvda-addon file.
+The URL should remain valid indefinitely.
+GitHub release URLs are recommended.
+
+Example: `"https://github.com/nvaccess/addon-datastore/releases/download/v0.1.0/myAddon.nvda-addon"`
+
+### sha256
+The SHA256 checksum for the *.nvda-addon file.
+To calculate a SHA256 sum on Windows, run the following from command prompt:
+
+```cmd
+certutil -hashfile <pathToAddonFile.nvda-addon> SHA256
+```
+
+Example: `"69D84CA8899800A5575CE31798293CD4FEBAB1D734A07C2E51E56A28E0DF8C82"`
+
+### sourceURL
+Allows reviewers to inspect the source code for common issues.
+
+Example: `"https://github.com/nvaccess/addon-datastore/"`
+
+### license
+The short name of the license
+
+Example: `"GPL v2"`
+
+### licenseURL
+Optional.
+A URL to the full license for the addon.
+
+Example: `"https://github.com/nvaccess/addon-datastore/license.MD"`
+
+### Metadata format
+For a full description of the schema see the
+[_validate/addonVersion_schema.json file](https://github.com/nvaccess/addon-datastore-validation/blob/main/_validate/addonVersion_schema.json).
+It includes an example of the file contents.
+
+## Submitting your add-on version file
+1. Create a PR to merge your branch into master on the `addon-datastore` repository
+1. Automated checks to validate the submission will complete.
+Refer to [addon-datastore-validation](https://github.com/nvaccess/addon-datastore-validation) for more information on automated checks.
+GitHub requires manual approval for the automated checks to run on an author's the first submission to the repository.
+1. If the checks pass, the PR should be merged automatically.
+However, a human review process may be required.
```

Comment:
When is human review required? Are you speaking of first author's submission as already written above? Or something else?